### PR TITLE
最新工時API - 多回傳 廠區/門市/分公司 (sector)欄位 

### PR DESCRIPTION
**Pull request by barry800414**
to fix: #114

### CODE REVIEW

File: test/testWorkings.js [hunk 1]
```diff
@@ -53,7 +55,12 @@ describe('Workings 工時資訊', function() {
                 .expect(200)
                 .expect(function(res) {
                     assert.deepPropertyVal(res.body.workings, '0.overtime_frequency', 4);
+                    assert.notDeepProperty(res.body.workings, '0.sector');
                     assert.notDeepProperty(res.body.workings, '0.author');
+                    assert.deepPropertyVal(res.body.workings, '1.sector', 'CCC');
+                    assert.notDeepProperty(res.body.workings, '2.author');
```
Comment (mark86092) [inline test/testWorkings.js]: Do you mean?

```
assert.notDeepProperty(res.body.workings, '2.sector');
```

File: routes/workings.js [hunk 2]
```diff
@@ -31,6 +31,7 @@ router.get('/latest', function(req, res, next) {
             week_work_time: 1,
             job_title: 1,
             overtime_frequency: 1,
+            sector: 1,
```
Comment (mark86092) [inline routes/workings.js]: remove the trailing space

**Comment by mark86092** [comment]
Please fix it

**Comment by barry800414** [comment]
I've fixed it

**Pull request merged**